script.js

```diff
@@ -0,0 +1,79 @@
+const $ = (selector) => document.querySelector(selector);
+const $$ = (selector) => document.querySelectorAll(selector)
+
+let currentCountries = {}
+
+const getAllCountries = async function () {
+  const response = await fetch('CountriesData.json')
+  return await response.json()
+}
+
+const displayCountries =  function (data) {
+  const countriesGrid = $('.countries-grid');
+  countriesGrid.innerHTML = ''
+
+  data.forEach(country => {
+    countriesGrid.innerHTML +=
+      `<a onClick="openDetailsPage('${country.name}')"  class="country scale-effect" data-country-name='${country.name}'>
+      <div class="country-flag">
+      <img src="${country.flag}" />
+      </div>
+
+      <div class="country-info">
+      <h2 class="country-title">${country.name}</h2>
+      <ul class="country-brief">
+        <li>
+        <strong>Population: </strong>
+        ${country.population.toLocaleString()}
+        </li>
+        <li>
+        <strong>Region: </strong>
+        ${country.region}
+        </li>
+        <li>
+        <strong>Capital: </strong>
+        ${country.capital}
+        </li>
+      </ul>
+      </div>
+    </a>`
+  });
+}
+
+const openDropDown = async function () {
+  $('.dropdown-body').classList.toggle("on")
+
```

Comment:
The name of the function indicate that this function should inly open the drop down. 

Then you toggle some class, maybe it will make more sense to call this function something like `toggleDropDown` ? 

Correct me if im wrong about what this function is doing